mivs/models.py

```diff
@@ -5,6 +5,16 @@ def href(url):
     return ('http://' + url) if url and not url.startswith('http') else url
 
 
```

Comment:
These properties are useful for both the ``IndieGame`` and ``IndieJudge`` models, so I pulled them out into a mixin.